cdap-authentication-clients/ruby/cdap-authentication-client.gemspec

```diff
@@ -18,7 +18,7 @@ require "cdap-authentication-client/version"
 
 Gem::Specification.new do |s|
   s.name        = "cdap-authentication-client"
-  s.version     = AuthenticationClient::VERSION
+  s.version     = CDAP::VERSION
   s.authors     = ["Yura Tolochkevych"]
```

Comment:
Hey Chris, while you're touching this code, I think it'd be a good idea to also change/remove human author names?
Instead, put CaskData Inc or something to that effect.